test/lifecycle/TimeFrame/TimeFrame.test.js

```diff
@@ -0,0 +1,144 @@
+const { time, shouldFail } = require('openzeppelin-test-helpers');
+const { expectBNIsWithinDelta } = require('../../helpers/expectBNIsWithinDelta');
+
+const TimeFrameMock = artifacts.require('TimeFrameMock');
+
+contract('TimeFrame', () => {
+  beforeEach(async () => {
+    this.now = await time.latest();
+
+    this.oneWeek = await time.duration.weeks(1);
+    this.twoWeeksAgo = await this.now - time.duration.weeks(2);
+    this.oneWeekAgo = this.now - this.oneWeek;
+    this.oneWeekFromNow = this.now + this.oneWeek;
+    this.twoWeeksFromNow = await this.now + time.duration.weeks(2);
+  });
+
+  describe('#hasStarted', () => {
+    it('it returns true when now if greater than start date', async () => {
```

Comment:
I'd personally prefer it if these tests were written in a way that they reflected how the return value of `TimeFrame` functions changes as time passes (e.g. using `time.increaseTo`), something like:

```
context before start
  it returns false

  context after start
    beforeEach
      time.advanceToStart

    it returns true
```

But that's just an idea, I won't block this PR on that :)